cfgov/v1/models/learn_page.py

```diff
@@ -106,6 +107,47 @@ class DocumentDetailPage(AbstractFilterPage):
     template = 'document-detail/index.html'
 
 
+class StoryPage(AbstractFilterPage):
```

Comment:
Oops, missed your first point in this comment.

I could, but it seemed logical to follow the pattern of declaring other subclasses of AbstractFilterPage in this same file. Should I break all of them out into their own file?